_content/index.md

```diff
@@ -1,3 +1,7 @@
+---
+layout: main.njk
```

Comment:
See comment above on `_content/about.md`.
This is all a bit workaround until a new version of Eleventy is published with that recent PR.